### ISSUE TITLE
Show Desktop applet no longer restores

### ISSUE THREAD

**Issue by Moilleadoir** [solus-project/budgie-desktop#1406]
**Operating System**: Solus 3.9995
**Budgie version**: budgie-desktop v10.4-55-g995e85e5

**The issue encountered**
The Show Desktop applet minimises all windows, but not longer restores them on a second click.

**Steps to reproduce the issue**
Add the Show Desktop applet to a panel.
Have windows visible
Click on the applet icon
Click again

**Comment by JoshStrobl** [comment]
Can confirm that this issue occurs, but only if the applet has already been added to the panel and you either restart budgie-panel or restart your system, not if it's newly added.

**Comment by EbonJaeger** [comment]
I've discovered while trying to debug this that when showing the desktop, something happens right after to toggle the mode off again. Almost by accident I found that when a tooltip is displayed (say, by hovering over the toggle desktop button for a second while clicking it) the show desktop mode is disabled without restoring the minimized windows.

Not sure at this point how to go about fixing that.
